assets/scss/common/top-bar.scss

```diff
@@ -422,10 +413,12 @@ $topbar-dropdown-arrows: false !default; //Set false to remove the \00bb >> text
             margin-top: 0;
             font-size: $topbar-back-link-size;
             a {
-              color: $topbar-link-color;
-              // line-height: ($topbar-height / 2);
+              background: $logo-color;
+              color: $lightest;
               display: block;
-              &:hover { background:none; }
+              &:hover, &:focus {
+                background: scale-color($logo-color, $lightness: -14%);
```

Comment:
I wonder if you could use a sass variable here for the `-14%` to ensure colours are scaled consistently?